cmi.proto

```diff
@@ -249,8 +249,16 @@ message GetMachineStatusRequest {
 }
 
 message GetMachineStatusResponse {
-    // Intentionally empty.
-    // Returns a non-error gRPC response if VM with matching machine object name is found.
+    // MachineID is the unique identification of the VM at the cloud provider.
+    // This could be the same/different from req.Name.
+    // MachineID typically matches with the node.Spec.ProviderID on the node object.
+    // Eg: gce://project-name/region/vm-machineID
+    // This field is REQUIRED.
+    string MachineID = 1;
+
+    // NodeName is the name of the node-object registered to kubernetes.
+    // This field is REQUIRED.
+    string NodeName = 2;
```

Comment:
can the driver/plugin retrieve the kubernetes node object? How does it get the node's name?
If the machine hasn't joined the cluster yet, what is the exptected behaviour then?

Reply:
The plugin will have to predict the node name. It is usually computable based on the provider type.

Reply:
This is in alignment with the create call response.